.gitignore

```diff
@@ -6,3 +6,13 @@ glue_data/
 user_config.sh
 .idea
 .ipynb_checkpoints/
+*.tgz
+*.sbatch
+commoncrawl.*
+subword-nmt/
+mosesdecoder/
+test-full/*
+prpn_logs
+prpn_logs/*
+sl*.out
+sl*.err
```

Comment:
Changing this file isn't a good idea unless there's a really clear need—leaving unexpected things in gitignore is a recipe for broken PRs later, and you shouldn't need to: Just use a working directory that's outside jiant/.